frontend/src/views/EventDetail.vue

```diff
@@ -160,6 +160,12 @@ export default {
     randomSortOrderIndicator() {
       if (this.sortBy != 'random') return ''
       return '◆'
+    },
+    countLiveQuestions() {
```

Comment:
This could be implemented as this.event.questions.length - this.countArchivedQuestions